app/models/concerns/foreman_remote_execution/host_extensions.rb

```diff
@@ -50,7 +50,7 @@ def remote_execution_proxies(provider)
                         proxy_scope = ::SmartProxy
                       end
 
-        proxies[:global] = proxy_scope.authorized.with_features(provider)
```

Comment:
I had to do this in order to ssh keys snippet to work with global proxy, I will probably open a separate PR and issue for this.